src/components/CodeModal.tsx

```diff
@@ -25,12 +25,7 @@ const CodeModal = ({ children, isOpen, setIsOpen, title }: CodeModalProps) => (
     <ModalOverlay />
     <ModalContent>
       <ModalHeader>{title}</ModalHeader>
-      <ModalCloseButton
-        style={{
-          right: "unset",
-          insetInlineEnd: "var(--eth-sizes-4)",
-        }}
-      />
+      <ModalCloseButton right="unset" insetInlineEnd="4" />
```

Comment:
Hmm this button's positioning looks to be working as expected on my end without these to props. And `insetInlineEnd` is already set the the Modal component theme.

Reply:
Okay, this was set with `style` in a previous PR, so I had updated to use style props instead, but I think changes since have corrected things so it's not necessary anymore. Good catch, will remove